server/helpers/toyProblemsChecker.js

```diff
@@ -0,0 +1,74 @@
+const githubQuery = require('./githubQuery');
+const { allCohorts } = require('../config/cohorts');
+const { allToyProblems } = require('../config/toyproblems');
+
+// so node won't throw an error and crash when a student doesn't have a fork
+process.on('uncaughtException', err => {
+  console.log('Caught exception: ', err);
+});
+
+const getStudentsList = cohortName => {
+  return allCohorts.filter(x => x.name === cohortName)[0].students;
```

Comment:
If the filter doesn't return anything it'll throw an error.

Refactor so it goes into a variable, check if there's something there before accessing with bracket notation